src/common/ecs-features/constants.ts

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+export const ECS_REQUEST_URL_TEMPLATE = "https://ecs.office.com/config/v1/PortalsMakerExperiences/1.0.0.0?AppName=powerpages-microsoft-com&EnvironmentID=${EnvironmentId}&UserID=${UserId}&TenantID=${TenantId}&region={Region}";
```

Comment:
This Url is specific to portals, please move it to Portals area instead of src/common

Reply:
this can be used for other products as well - making it more generic